packages/cli/commands/add/workspace.ts

```diff
@@ -23,6 +23,8 @@ export async function createWorkspace({
 	const viteConfigPath = path.join(resolvedCwd, commonFilePaths.viteConfigTS);
 	let usesTypescript = fs.existsSync(viteConfigPath);
```

Comment:
I do wonder why we chose to detect typescript based on the vite config file in the past. But since it's working i'm not going to touch it.